backend/models/Locations.ts

```diff
@@ -5,27 +5,35 @@ type ratingType = {
   dislikes: string[];
 };
 
+type comentType = {
+  author: string;
+  text: string;
+  likes: string[];
+  dislikes: string[];
+};

```

Comment:
we discussed today that there should be a createdAt field in each comment

also, coMMent - correct